sum-of-multiples.md

```diff
@@ -4,4 +4,12 @@ multiples of either 3 or 5, we get 3, 5, 6 and 9, 10, 12, 15, and 18.
 The sum of these multiples is 78.
 
 Write a program that can find the sum of the multiples of a given set of
-numbers. If no set of numbers is given, default to 3 and 5.
+numbers.
+
+This exercise is in the process of being reworked.
```

Comment:
Will this scare people off into thinkin "maybe I shouldn't do this exercise"?

If so, I should explicitly say "You can still do this exercise and submit solutions, but the test cases may differ from the README"